src/XMakeTasks/Microsoft.Common.Traversal.targets

```diff
@@ -0,0 +1,16 @@
+﻿<!--
+***********************************************************************************************
+Microsoft.Common.Traversal.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (C) Microsoft Corporation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(CoreTraversalTargetsPath)" 
```

Comment:
I will define it in our after-props hook. We are only redirecting targets, not props so it will work. We need to discuss the outer build extensibility and how nuget packages participate, etc. This is just enough to (1) unblock my follow up work (2) smoke test that the common targets hijacking doesn't interfere with existing projects.